### DISCUSSION TITLE
Cannot apply for Github Student plan after making new account.

### DISCUSSION THREAD

**Discussion by UwUCatGirlMeow** [community/community#38173] [GitHub Education]
I make a new github account, and switched my school. But when I apply I get this error.
![image](https://user-images.githubusercontent.com/116464156/199749198-05c92a04-40fe-41a8-bba9-8ef83b1fb78c.png)

**Comment by samus-aran** [comment]
Hi @UwUCatGirlMeow 👋 

You will need to speak to your school and enquiry how another student is using that email if it has been allocated to you.
